README.md

```diff
@@ -13,9 +13,7 @@ compare and validate key usage across multiple language versions.
 By identifying inconsistencies between these translations, `linguist` ensures
 that they remain accurate and complete.
 
-## Usage
-
-### Cloning Repositories with SSH Authentication
+## Prerequisites
 
 To run `linguist`, you must specify the **path to your SSH private key** as an
```

Comment:
The part about the SSH key execution option is already sufficiently explained in the Usage section, so I suggest removing it to avoid redundancy.